HerPublicWebsite.BusinessLogic/Models/Enums/IncomeBand.cs

```diff
@@ -14,5 +14,9 @@ public enum IncomeBand
     [GovUkRadioCheckboxLabelText(Text = "£34,500 or less")]
     UnderOrEqualTo34500,
     [GovUkRadioCheckboxLabelText(Text = "More than £34,500")]
-    GreaterThan34500
+    GreaterThan34500,

```

Comment:
We should add a comment somewhere here to explain that the top four are no longer used, but kept for the purposes of preserving historic data. You might be able to mark them as `[Obsolete("Some helpful explanation")]`, so that the IDE will warn about them (we'll then need to suppress the warning in a couple of places though: might not be worth it depending on how many)